### DISCUSSION TITLE
past tense for situations vs present tense for events?

### DISCUSSION THREAD

**Discussion by HuibertDeVries** [nemo-ufes/gufo#35] [Q&A] [answered]
What is the reason that gufo properties related to Situations are in present tense, while gufo properties related to Events are in past tense?

**Comment by pedropaulofb** [comment]
Dear @HuibertDeVries ,

I believe this is because in UFO/OntoUML "_events are locked in the past_" [(check it here)](https://www.researchgate.net/profile/Giancarlo-Guizzardi-2/publication/334330316_Events_as_Entities_in_Ontology-Driven_Conceptual_Modeling/links/5d24d751a6fdcc2462d05551/Events-as-Entities-in-Ontology-Driven-Conceptual-Modeling.pdf).

**Comment by HuibertDeVries** [comment]
Hi @pedropaulofb ,

Thank you for your answer!

It leaves me with two follow-up questions.

1. I indeed assumed something like that. However, Events may have started but not yet ended. Like a soccer game that is still going on. On the other hand Situations may have been finished in the past (expressed by hasEndPoint). So in that respect I see no difference between these two. Why is a distinction made between between the properties related to Events and those related to Situations?
2. Furthermore, if Events are locked in the past, how do I model ongoing Events?

**Comment by pedropaulofb** [comment]
Hi @HuibertDeVries,

These are indeed good questions that I do not have an answer. I will leave it to @jpalmeida =)

However, I know that a workaround to deal with 'future events' is to use types. In this case, we can that say a Person may have a relation (worksIn ?) with Job Type <<Type>> to represent that persons may have future (still unknown) jobs. You can see this strategy in use in [this paper here](https://research.utwente.nl/files/300281323/978_3_031_17995_2_26.pdf).

**Comment by HuibertDeVries** [comment]
Hi Pedro,

Thank you for your explanation so far!

For both other questions i'll wait for @jpalmeida answer.

**Comment by jpalmeida** [comment]
Hi @HuibertDeVries, 

The fact that there is no statement for hasEndPoint in a knowledge base, does not mean the event is ongoing, because of the open-world assumption.

But, regardless of this, even if you want to admit ongoing events, the past tense in the verbs (gufo:wasCreatedIn, gufo:wasTerminatedIn) refers to effects established in the past, not to the event being in the past.

**Comment by ManuelSimoes** [comment]
I assumed the past tense referred to the fact that you cannot know an Event fully until it's complete. Something that unfolds in time is only a whole after it's finished unfolding?

**Reply by jpalmeida** [reply, marked as answer]
As Pedro commented earlier, the UFO position is that events only exist in the past. So, an event strictly speaking cannot gain new parts in time, and cannot be an ongoing event. 

However, I was entertaining the possibility that someone could admit ongoing events (which is something that Huibert seemed to be interested in, and which has been explored recently by Giancarlo Guizzardi and Nicola Guarino [1]). (There's nothing in gUFO that prevents someone from interpreting an event as ongoing.) My point is that, even in that case, the past tense is justified, not by the event being finished, but by the fact that the effects described by the object properties always happened in the past (that something was created, was terminated, participated, even in an ongoing event). 

[1] http://dx.doi.org/10.1007/s11229-024-04505-2

**Reply by HuibertDeVries** [reply]
Thank you for your response @jpalmeida, this is clarifying!
